system/settings/settings.xml

```diff
@@ -1411,7 +1411,7 @@
         </setting>
         <setting id="pvrplayback.confirmchannelswitch" type="boolean" label="19281" help="36231">
           <level>1</level>
-          <default>false</default>
+          <default>true</default>
```

Comment:
okay, i try to by more clear:

before:
- 1-9 opens numeric input dialog
- setting influences whether dialog auto closes or not

after:
- 1-9 does not open dialog, only dispalys the entered number
- setting does not have an affect now
- always auto-switching

short: the setting we're discussing here has nothing to do with channel switching via numeric input anymore

so, no problem.